Hunted Mobile/Hunted Mobile/Hunted Mobile/Repository/ErrorRepository.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Hunted_Mobile.Service;
+
+using PusherClient;
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Hunted_Mobile.Repository {
+    public class ErrorRepository {
+        public async void Create(Exception e) {
+            var w32ex = e as Win32Exception;
+            int errorCode = 137;
+
+            if(w32ex == null) {
+                if(e.InnerException != null) {
+                    w32ex = e.InnerException as Win32Exception;
+                    errorCode = w32ex.ErrorCode;
+                }
+            }
```

Comment:
```suggestion
            if(w32ex == null && e.InnerException != null) {
                    w32ex = e.InnerException as Win32Exception;
                    errorCode = w32ex.ErrorCode;
            }
```
Volgensmij Stackoverflow code 😂